kazoo/client.py

```diff
@@ -594,7 +597,10 @@ def stop(self):
             return
 
         self._stopped.set()
-        self._queue.append((CloseInstance, None))
+        if self._keep_session:
+            self._queue.append((None, None))
```

Comment:
Would it be better to use another named constant, instead of None here, something a little more obvious as to what this is doing?